tests/drivers/clock_control/stm32_clock_configuration/stm32h7_core/src/test_stm32_clock_configuration.c

```diff
@@ -75,5 +75,18 @@ ZTEST(stm32_syclck_config, test_pll_src)
 			RCC_PLLSOURCE_NONE, pll_src);
 #endif
 
+}
+
+ZTEST(stm32_syclck_config, test_hse_css)
+{
+#if STM32_HSE_ENABLED
```

Comment:
Nitpick: Put this #if around the full `ZTEST(stm32_syclck_config, test_hse_css)`